doc/manual.md.in

```diff
@@ -13,3 +13,7 @@ builders.md
 development.md
 contributing.md
 ```
+
+```{=include=} appendix
+platform-support.appendix.md
+```
```

Comment:
I feel like this really shouldn't need to be in an appendix. This is fairly important information and should probably be somewhere in the beginning of the manual. Could you look around to see if it would fit somewhere else?

Reply:
Right, although it's RFC046 that stated this should be an appendix:

> The definition of support tiers and a list of platforms with the corresponding support tiers is added as an appendix to the Nixpkgs manual. 

Not really sure how binding that is. And yes, we're not exactly implementing that RFC here, but assuming one day it will be implemented (unlikely IMO), and assuming it's binding, then we would have to relocate the section to an appendix, which would be bad.

This also makes me wonder: if we expect the content to move, then the anchor should not contain the preifx `appendix`.

Reply:
Maybe have a short description in the main body, and copy the tier definitions into appendix?

Reply:
Yes, support information should be in one of the first sections in the manual.

Reply:
Yeah, and for now we don't need any lengthy details. So I think this entire section should just be somewhere in the beginning of the manual instead.

Reply:
Sounds good, I'll look for a place. I do think we need to make this section a little more discursive if it's not tucked in in an appendix. Don't worry, I'll leave the table in ;)